container-guide/adoc/containers-podman-overview.adoc

```diff
@@ -93,31 +110,133 @@ command. Then use the following command to change the group ownership and rights
+files belong to different user IDs in the container and on the host. {podman} can remediate this
+catch using the command line flag `--userns=keep-id`. With this flag {podman} will keep your current
+user id in the container:
+
+....
+❯ podman run --userns=keep-id --rm -it registry.suse.com/bci/bci-base id
+uid=1000(user) gid=1000(users) groups=1000(users)
+....
+
+The flag `--userns=keep-id` has a similar effect when used with bind mounts:
+
+....
+ ❯ podman run --rm -it -v $(pwd):/share/ registry.suse.com/bci/bci-base stat /share/
+  File: /share/
+  Size: 318             Blocks: 0          IO Block: 4096   directory
+Device: 2ch/44d Inode: 3506170     Links: 1
+Access: (0755/drwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: 2023-05-03 12:52:18.636392618 +0000
+Modify: 2023-05-03 12:52:17.178380923 +0000
+Change: 2023-05-03 12:52:17.178380923 +0000
+ Birth: 2023-05-03 12:52:15.852370288 +0000
+
+ ❯ podman run --userns=keep-id --rm -it -v $(pwd):/share/ registry.suse.com/bci/bci-base stat /share/
+  File: /share/
+  Size: 318             Blocks: 0          IO Block: 4096   directory
+Device: 2ch/44d Inode: 3506170     Links: 1
+Access: (0755/drwxr-xr-x)  Uid: ( 1000/     user)   Gid: ( 1000/     users)
+Access: 2023-05-03 12:52:18.636392618 +0000
+Modify: 2023-05-03 12:52:17.178380923 +0000
+Change: 2023-05-03 12:52:17.178380923 +0000
+ Birth: 2023-05-03 12:52:15.852370288 +0000
+....
+
+{podman} stores your containers data in the storage graph root (by default
+`~/.local/share/containers/storage`). It is possible that there are files inside the graph root not
+owned by your user due to the way {podman} maps non-root users in containers. Files owned by such a
+remapped non-root user are often not readable by your current user.
+
+In case you have to read or modify any files in the graph root, you can enter a shell with
+privileges to do so as follows:
+
+....
+ ❯ podman unshare bash
+
+ ❯ id
+uid=0(root) gid=0(root) groups=0(root),65534(nobody)
+....
+
+Note that `podman unshare` performs the same user remapping as `podman run` does when launching a
+rootless container. You cannot gain elevated privileges via `podman unshare`.
+
+
+==== Caveats of rootless containers
+
+Rootless containers currently suffer a substantial performance penalty for network transfers. This
+penalty is caused by the inability of unprivileged users to configure network namespaces on
+Linux. Therefore {podman} relies on a userspace network implementation called `slirp4netns`. This
+userspace networking has to emulate the full TCP-IP stack and can cause a heavy performance
+degradation for workloads relying on high network transfer rates.
+
+On Linux unprivileged users cannot open ports below port number 1024. This limitation also applies
+to {podman} and therefore rootless containers can by default not expose ports below port number
+1024. You can lift this limitation via:
```

Comment:
```suggestion
Because unprivileged users cannot configure network namespaces on
Linux, {podman} relies on a userspace network implementation called `slirp4netns`. It emulates the full TCP-IP stack and can cause a heavy performance degradation for workloads relying on high network transfer rates. This means that rootless containers suffer from slow network transfers.

On Linux, unprivileged users cannot open ports below port number 1024. This limitation also applies
to {podman}, so by default, rootless containers cannot expose ports below port number
1024. You can remove this limitation using the following command:
```